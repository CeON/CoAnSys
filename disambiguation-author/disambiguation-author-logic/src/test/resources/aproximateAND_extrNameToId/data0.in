c1	Kowalski	[A#{(An Empirical Study on the AND)},7#{(2011)}]
