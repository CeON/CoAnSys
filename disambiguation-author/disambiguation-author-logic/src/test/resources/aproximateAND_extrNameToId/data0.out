({(c1,Kowalski,[7#{(2011)},A#{(An Empirical Study on the AND)}])},{})
